packages/blocks/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+import { PlusIcon, SearchIcon } from '@mergestat/icons';
+import { Tag } from '../Filter/Tag';
+import { Checkbox } from '../Form';
+
+type MultiSelectProps = {
+  setStateToProps: {
+    title: string;
+    checked: boolean;
+  }[];
+  getState?: ([]) => void;
+};
+
+export const MultiSelect: React.FC<
+  MultiSelectProps &
+    React.DetailedHTMLProps<
+      React.DetailsHTMLAttributes<HTMLElement>,
+      HTMLElement
+    >
+> = ({ setStateToProps, getState }) => {
+  const [state, setState] = useState(setStateToProps);
+  const [value, setValue] = useState('');
+  const [isActive, setIsActive] = useState(false);
+
+  window.addEventListener('click', (e) => {
+    const target = e.target as Element
+    if (target.localName !== 'input' && target.localName !== 'label') {
+      setIsActive(false);
+    }
+  });
+
+  window.addEventListener('keypress', (e) => {
+    if (e.key === 'Enter' && value.length > 0) {
+      setState([...state, { title: value, checked: true }]);
+
+      if (getState) {
+        getState([...state]);
+      }
+      setValue('');
+    }
+  });
```

Comment:
I'm seeing odd behavior when i click on the item _label_ it closes out the pop-up select box:

https://user-images.githubusercontent.com/57259/163187138-4a6e5ebe-a55b-44b3-aac2-b4676819c0f6.mov